Aplib.Core/Desire/GoalStructures/FirstOfGoalStructure.cs

```diff
@@ -37,38 +38,48 @@ public FirstOfGoalStructure(IMetadata metadata, params IGoalStructure<TBeliefSet
         public override IGoal<TBeliefSet> GetCurrentGoal(TBeliefSet beliefSet)
             => _currentGoalStructure!.GetCurrentGoal(beliefSet);
 
-        /// <inheritdoc />
+        /// <summary>
+        /// This method updates the status of the <see cref="FirstOfGoalStructure{TBeliefSet}" />.
+        /// The goal structure status is set to:
+        /// <list type="table">
+        /// <item>
+        /// <term><see cref="Success"/></term>
+        /// <term>When any one of its children is successful.</term>
+        /// </item>
+        /// <item>
+        /// <term><see cref="Failure"/></term>
+        /// <term>When all children fail.</term>
+        /// </item>
+        /// <item>
+        /// <term><see cref="Unfinished"/></term>
+        /// <term>Otherwise.</term>
+        /// </item>
+        /// </list>
+        /// </summary>
```

Comment:
- Added indentation for readability.
- Fixed the tag types, the second one of the item should be `description`.
- Changed the list type to bullet, this looks better in both docfx and Rider, especially for the RepeatGoalStructure one.

Reply:
Great suggestions. For 3, however, I tried both _bullet_ and _table_, but think _table_ is more clear when combined with 2 (_description_ instead of _term_).